spec/aruba/spawn_process_spec.rb

```diff
@@ -56,5 +56,35 @@
 
       it { expect {process.start}.to raise_error Aruba::LaunchError }
     end
+
+    context "with a childprocess launch error" do
+      let(:child) { instance_double(ChildProcess::AbstractProcess) }
```

Comment:
I know, but don't fiddle with someone else's classes. Why not using this instead: https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles/using-a-class-double